sass/_Variables.scss

```diff
@@ -16,6 +16,7 @@ $color-vibrant-blue: #009ddc;
 $color-teal: #296896;
 $color-greyish-cyan: #cddee9;
 $color-greyish-light-cyan: #cddee9;
+$color-light-contrast-grey: #7e8c9a;
 $color-light-grey: #bcc3ca;
```

Comment:
#7e8c9a = hsl(210, 12%, 55%) (contrast against white of 3.43:1)
#bcc3ca = hsl(210, 12%, 76%) (contrast against white of 1.78:1)